client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -305,6 +308,32 @@ class ManageMenu extends PureComponent {
 					return memo;
 				}
 
+				//Special handling for feedback (contact form entries), let's calypsoify except for VIP
+				//It doesn't make sense for the author to use the generic CPT handling in Calypso
+				if ( postTypeSlug === 'feedback' ) {
+					return memo.concat( {
+						name: postType.name,
+						label: decodeEntities( get( postType.labels, 'menu_name', postType.label ) ),
+						config: 'manage/custom-post-types',
+						//controls if we show the wp-admin link. It feels like this is coupling two different meanings (api_queryable)
+						queryable: false,
```

Comment:
I could update the feedback API response `postType.api_queryable` to return false, but I'm not actually sure what this value is supposed to represent, or what this might effect if I do so. It seems odd that this is tied to whether wp-admin links display.